google-calendar-events.php

```diff
@@ -60,3 +60,22 @@
 
 // Load plugin.
 include_once 'includes/main.php';
+?>
+<link href="/css/output-style.css" rel="stylesheet">
```

Comment:
This is not how to load styles in WordPress. Also these styles should only be loaded where it is required, currently it would load even in frontend where it is actually not required.

Reply:
i will load on in that setting page.